packages/accounts/src/account-header.ts

```diff
@@ -0,0 +1,13 @@
+import { Address } from '@solana/addresses';
+import { LamportsUnsafeBeyond2Pow53Minus1 } from '@solana/rpc-types';
+
+/** The amount of bytes required to store the account header. */
+export const ACCOUNT_HEADER_SIZE = 128;
+
+/** Describe the generic account details applicable to every account. */
+export type AccountHeader = {
+    readonly programAddress: Address;
+    readonly executable: boolean;
+    readonly lamports: LamportsUnsafeBeyond2Pow53Minus1;
+    readonly rentEpoch?: bigint;
```

Comment:
Oh I didn't know that. Someone asked me about that field in Umi recently though (cause it's wrongly a number there instead of a bigint) so it seems some people are still using it?

Reply:
Not for long.

Reply:
I'm not familiar enough with it to be honest. If you say this can be removed, I'm happy to remove it! 👌

Reply:
Removed ✅